CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to this project will be documented in this file.
 
-## \[Unreleased\]
```

Comment:
A new unreleased section is required, since the changes for 2.4 are gradually arriving